Modules/_ctypes/_ctypes.c

```diff
@@ -5476,18 +5476,24 @@ comerror_init(PyObject *self, PyObject *args, PyObject *kwds)
     return 0;
 }
 
+static int
+comerror_clear(PyObject *self) {
+    return ((PyTypeObject *)PyExc_BaseException)->tp_clear(self);
+}
+
 static int
 comerror_traverse(PyObject *self, visitproc visit, void *arg)
 {
     Py_VISIT(Py_TYPE(self));
-    return 0;
+    return ((PyTypeObject *)PyExc_BaseException)->tp_traverse(self, visit, arg);
 }
 
 static void
 comerror_dealloc(PyObject *self)
 {
     PyTypeObject *tp = Py_TYPE(self);
     PyObject_GC_UnTrack(self);
+    (void)comerror_clear(self);
```

Comment:
Why is this `(void)` necessary?

Reply:
Good questuin. Sadly, I don't know why is this necessary.
Other `_dealloc` functions follows it:
https://github.com/python/cpython/blob/080a5961527473af182b025bb29e0c52d43fd49e/Modules/_ctypes/callproc.c#L499-L507

Reply:
Because `PyCArg_clear` returns `int`, some tools might report "Unused return value" without an explicit `(void)` cast.

Reply:
Yes, as Nikita says, some compilers or linters may complain about unused result values, but there's (IMO) great value in explicitly communicating to those who're reading your code that:

- yes, I know this function is returning a non-void value
- yes, I deliberately do not care about that resulting value

Without the `void` cast, you can easily think you're looking at code that lacks error handling.